tests/pre_merge/models/test_feature_extractor.py

```diff
@@ -1,7 +1,11 @@
 import pytest
 import torch
+from torchvision.models.efficientnet import EfficientNet_B5_Weights
 
-from anomalib.models.components.feature_extractors import FeatureExtractor
+from anomalib.models.components.feature_extractors import (
+    TimmFeatureExtractor,
+    get_torchfx_feature_extractor,
```

Comment:
Would it be an idea to have `TorchFeatureExtractor` just for the naming consistency with `TimmFeatureExtractor`? It would use `get_torchfx_feature_extractor` in its `__call__`. Or would it be an overkill?